policies/cis_gcp_foundations_benchmark.yaml

```diff
@@ -0,0 +1,11 @@
+id: d8792397-239e-4028-9539-7b3771ba929f
+policyId: CIS GCP
+policyName:  CIS Google Cloud Platform Foundation
+description: >
+  The CIS Security Benchmarks program provides well-defined, unbiased, consensus-based
+  industry best practices to help organizations assess and improve their security
+enabled: true
+rules:
+  - gcp_service_account_no_user_managed_keys.yaml
+  - gcp_IAM_users__not_assigned_the_Service_Account_User_or_Service_Account_Token_Creator _roles at _level.yaml
```

Comment:
file name is bad. should be shorter, and without spaces

Reply:
What filename you propose here?

Reply:
hmm. e.g. `no_project_level_service_account_user_or_token_for_iam_user.yaml'